### PR TITLE
R4 Practitioner updates

### PR DESCRIPTION
**Pull request by amcginn**
Description
----
* Removed extra search examples
* Changed 'id' to 'ID' throughout
* Fields
  * Updated text
  * image
  * image
* Moved disclaimer and errors (see above)
* Extension
  * Updated description
  * image
  * image
* Search parameters
  * Updated formatting and descriptions
  * Updated notes
  * image
  * image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by RicardoCrespo-ora — APPROVED** [review]

File: content/millennium/r4/base/individuals/practitioner.md [hunk 1]
```diff
@@ -19,31 +19,33 @@ Not all practitioners have access to the EHR but can be referenced by other reso
...
+## Custom Extensions
 
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
 
- ID                    | Value\[x] Type                                              | Description
------------------------|----------------|------------------------------------------------------------------------------------------------
- `is-physician`        | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean) | Indication of whether the provider is a physician or not.
+| ID             | Value\[x] Type                                              | Description                                                   |
+|----------------|-------------------------------------------------------------|---------------------------------------------------------------|
+| `is-physician` | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean) | Indication of whether the practitioner is a physician or not. |
```
Comment (kckeim) [inline content/millennium/r4/base/individuals/practitioner.md:48]: You made the `boolean` link at the end of the markdown file - you can use it here

File: lib/resources/r4/practitioner.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ fields:
     {
       "resourceType": "Practitioner"
     }
-  note: resourceType must be Practitioner.
+  note: resourceType must be `Practitioner`.
   url: http://hl7.org/fhir/R4/json.html#resources
```
Comment (kckeim) [inline lib/resources/r4/practitioner.yaml:15]: https-ify, please.

**Review by kckeim — APPROVED** [review]
Good edits overall - I appreciated getting to pre-review with you over Slack.
Nitpick adds from me is all, if you want to make them.

**Review by cldeitrich — APPROVED** [review]
[OHAIUAD-204312](https://jira3.cerner.com/browse/OHAIUAD-204312): Approved for style.

File: lib/resources/r4/practitioner.yaml (hunk 2 above)
Comment (amcginn) [reply lib/resources/r4/practitioner.yaml:15]:
```suggestion
  url: https://hl7.org/fhir/R4/json.html#resources
```

File: content/millennium/r4/base/individuals/practitioner.md [hunk 3]
```diff
@@ -9,45 +9,47 @@ title: Practitioner | R4 API
...
+## Custom Extensions
 
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
 
- ID                    | Value\[x] Type                                              | Description
------------------------|----------------|------------------------------------------------------------------------------------------------
- `is-physician`        | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean) | Indication of whether the provider is a physician or not.
+| ID             | Value\[x] Type                                              | Description                                                   |
+|----------------|-------------------------------------------------------------|---------------------------------------------------------------|
+| `is-physician` | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean) | Indicates whether the practitioner is a physician. |
```
Comment (amcginn) [inline content/millennium/r4/base/individuals/practitioner.md:48]:
```suggestion
| `is-physician` | [`boolean`] | Indicates whether the practitioner is a physician. |
```

**Comment by shushruth-s** [comment]
After Deployment Screenshots
Screenshot 2024-02-02 at 2 05 56 PM
Screenshot 2024-02-02 at 2 06 35 PM
Screenshot 2024-02-02 at 2 07 42 PM

**Pull request merged**